README

Znunu opened the pull request:
Mostly slight changes to change the tone. The part about the discord warning, I completely changed. Frankly, I can't imagine anyone that knows JS would use this script, they'd make their own. The 'If you're a risk taker' just makes it seem like we're ourselves saying it's a scam??? So everyone else won't use it either.

Znunu committed:
Update README.md

Multarix commented:
I'll merge this when I get home, however I'd like to point out that the reason they have a giant message saying anything is likely a scam, is due to your user token technically being able to be obtained and or sent to another person via the console if you pasted in a harmful script, which could result in your account being hacked, so it's a valid warning, and me being up front about it should make a user at least consider that not all scripts are going to trustworthy, and think twice. It basically just serves as an extra warning.

Multarix committed:
Changed wording for console warning

This should work as a better reason and explain to non-programmy people why those warnings are there.

Multarix commented:
I changed the console warning to be more in line of proper reasons. Let me know if it should be changed further, for now I'll accept this pull request.

Znunu commented:
Alright, this is perfect. Thanks for merging.
